File: Problem Statements/Greatest of 3 Numbers/Go2N.java
import java.util.Scanner;
class Go2N{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter 1st Number : ");
        int a=sc.nextInt();
        System.out.print("Enter 2nd Number : ");
        int b=sc.nextInt();
        System.out.print("Enter 3rd Number : ");
        int c=sc.nextInt();
        sc.close();
        if(a!=b && b!=c && a!=c){
            if(a>b && a>c){
               System.out.println(a+" is the Greatest Number ");
            }else if(b>a && b>c){
                System.out.println(b+" is the Greatest Number ");
            }else{
                System.out.println(c+" is the Greatest Number ");
            }
        }else{
            System.out.println("Try with 3 different Numbers");
        }
    }
}

File: Problem Statements/Greatest of 3 Numbers/Go2N.c
#include<stdio.h>
void main(){
    int a,b,c;
    printf("Enter 1st Number : ");
    scanf("%d",&a);
    printf("Enter 2nd Number : ");
    scanf("%d",&b);
    printf("Enter 3rd Number : ");
    scanf("%d",&c);
    if(a!=b && b!=c && a!=c){
        if(a>b && a>c){
            printf("%d is the Greatest Number ",a);
        }else if(b>a && b>c){
            printf("%d is the Greatest Number ",b);
        }else{
            printf("%d is the Greatest Number ",c);
        }
    }else{
        printf("Try with 3 different Numbers");
    }
}